src/locales/de/battle.ts

```diff
@@ -55,5 +55,9 @@ export const battle: SimpleTranslationEntries = {
   "skipItemQuestion": "Bist du sicher, dass du kein Item nehmen willst?",
   "notDisabled": "{{pokemonName}}'s {{moveName}} ist\nnicht mehr deaktiviert!",
   "eggHatching": "Oh?",
-  "ivScannerUseQuestion": "IV-Scanner auf {{pokemonName}} benutzen?"
+  "ivScannerUseQuestion": "IV-Scanner auf {{pokemonName}} benutzen?",
+  // "useMove": " used\n{{moveName}}!",
+  // "fainted": " fainted!",
+  // "foe": "Foe",
```

Comment:
```suggestion
  // "foe": "(Gegner)",
```

(This needs to be AFTER the Pokemon name in german!)

See https://github.com/pagefaultgames/pokerogue/pull/1276